chainntnfs/txnotifier.go

```diff
@@ -1507,6 +1507,13 @@ func (n *TxNotifier) handleConfDetailsAtTip(confRequest ConfRequest,
 
 	// TODO(wilmer): cancel pending historical rescans if any?
 	confSet := n.confNotifications[confRequest]
+
+	// If we already have details for this request, we don't want to add it
+	// again since we have already dispatched notifications for it.
+	if confSet.details != nil {
+		return
```

Comment:
add a logging statement?

Reply:
Done. Hope the wording and log level is adequate.